app/views/requests/request_mailer/_item_info_all.html.erb

```diff
@@ -0,0 +1,27 @@
+<div style="margin-bottom:18px;font-weight:normal;font-size:16px;">Items Requested</div>
+<table style="width:100%;">
+  <% @submission.items.each_with_index do |item, index| %>
+  <tr>
+    <% unless item[:selected].empty? %>
+    <% recalled = (item[:type] == "recall") ? true : false %></td>
+    <td style="border:1px solid #ccc;padding: 18px;">
+      <table width="100%" cellpadding="0" cellspacing="0" border="0" class="responsive-table">
+        <tr>
+          <% item.each do |key, value| %>
+          <% unless key == "selected" || value.empty? %>
+          <td valign="top" class="responsive-column-container-all">
+            <div style="font-weight:normal;font-size:12px;">
+              <% if key == "pickup" %>Pick-up<% else %><%= format_label(key) %><% end %></div>
```

Comment:
Would it be better to just have this as the first row of the table, and then just skip it in the loop?

Reply:
I don't think it would display full-width on mobile if we did that.